subprojects/assemble/build-metrics/src/main/kotlin/com/avito/android/plugin/build_metrics/internal/runtime/Cgroup2.kt

```diff
@@ -40,12 +40,18 @@ internal sealed class Cgroup2 {
             val cgroupDir = File("/sys/fs/cgroup")
 
             return if (cgroupDir.exists()) {
-                Available(
-                    memory = Memory(
-                        memoryMax = File(cgroupDir, "memory.max"),
-                        memoryCurrent = File(cgroupDir, "memory.current")
+                val memoryMax = File(cgroupDir, "memory.max")
+                val memoryCurrent = File(cgroupDir, "memory.current")
+                if (memoryMax.exists() && memoryCurrent.exists()) {
```

Comment:
As I see now we never find this files at CI build.
I check locally inside docker container that files exists.
So I have no idea why we can't find them at CI builds

Reply:
I have only one idea is try to log files inside directory `/sys/fs/cgroup` at CI to see what are there